packages/edit-site/src/components/page-pages/index.js

```diff
@@ -161,31 +161,40 @@ function FeaturedImage( { item, viewType } ) {
 		canvas: 'edit',
 	} );
 	const hasMedia = !! item.featured_media;
+	const size =
+		viewType === LAYOUT_GRID
+			? [ 'large', 'full', 'medium', 'thumbnail' ]
+			: [ 'thumbnail', 'medium', 'large', 'full' ];
 	return (
 		<span
 			className={ {
 				'edit-site-page-pages__media-wrapper':
 					viewType === LAYOUT_TABLE,
 			} }
 		>
-			<button
-				className="page-pages-preview-field__button"
-				type="button"
-				onClick={ onClick }
-				aria-label={ item.title?.rendered || __( '(no title)' ) }
-			>
-				{ hasMedia && (
-					<Media
-						className="edit-site-page-pages__featured-image"
-						id={ item.featured_media }
-						size={
-							viewType === LAYOUT_GRID
-								? [ 'large', 'full', 'medium', 'thumbnail' ]
-								: [ 'thumbnail', 'medium', 'large', 'full' ]
-						}
-					/>
-				) }
-			</button>
+			{ viewType === LAYOUT_LIST && hasMedia && (
+				<Media
+					className="edit-site-page-pages__featured-image"
```

Comment:
Nit: I guess we could have `Media` in a constant above as it's identical in both scenarios